polaris-react/src/components/DropZone/DropZone.tsx

```diff
@@ -390,24 +391,28 @@ export const DropZone: React.FunctionComponent<DropZoneProps> & {
         action={labelAction}
         labelHidden={labelHiddenValue}
       >
-        <div
+        <Box
```

Comment:
What you could try here is separating the visual styles from the interactive functionality by having both the `Box` for the visual and layout styles and having the `div` for the interactive functionality.